pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/SegmentStatusChecker.java

```diff
@@ -301,7 +325,11 @@ public void setMetricsRegistry(ControllerMetrics metricsRegistry) {
     _metricsRegistry = metricsRegistry;
   }
 
-  void setStatusToDefault() {
+  public void setTableSizeReader(TableSizeReader tableSizeReader) {
```

Comment:
Do we need a setter for this?

Reply:
The usage of `setTableSizeReader` is just the same as the one of `setMetricsRegistry`.